code/game/gamemodes/shadowling/shadowling_items.dm

```diff
@@ -0,0 +1,69 @@
+/obj/item/clothing/under/shadowling
```

Comment:
This file should not exist; the shadowling species should have all of the effects of these items.